modules/openapi-generator/src/main/resources/JavaSpring/model.mustache

```diff
@@ -46,7 +46,7 @@ import javax.annotation.Generated;
 {{>enumOuterClass}}
 {{/isEnum}}
 {{^isEnum}}
-{{>pojo}}
+{{#vendorExtensions.x-is-one-of-interface}}{{>oneof_interface}}{{/vendorExtensions.x-is-one-of-interface}}{{^vendorExtensions.x-is-one-of-interface}}{{>pojo}}{{/vendorExtensions.x-is-one-of-interface}}
```

Comment:
Is this still necessary after https://github.com/OpenAPITools/openapi-generator/pull/11784 ?